go/cmd/dolt/commands/remote.go

```diff
@@ -207,21 +225,97 @@ func parseRemoteArgs(apr *argparser.ArgParseResults, scheme, remoteUrl string) (
 	return params, nil
 }
 
-func printRemotes(dEnv *env.DoltEnv, apr *argparser.ArgParseResults) errhand.VerboseError {
-	remotes, err := dEnv.GetRemotes()
+func callSQLAdd(sqlCtx *sql.Context, queryist cli.Queryist, remoteName, remoteUrl string) error {
```

Comment:
Given that the function `callSQLAdd` is visible to the entire `commands` package, we should give it a more specific name like `callSQLAddRemote`.